jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/jdt/jax_rs/Jax_RSConstants.java

```diff
@@ -21,18 +21,25 @@
 public class Jax_RSConstants {
 
     public static final String RESOURCE_METHOD = "ResourceMethod";
-    
+
     /* Annotation Constants */
-    public static final ArrayList<String> METHOD_DESIGNATORS = new ArrayList<String>(List.of(
-            "Path", "GET", "POST", "PUT", "DELETE", "HEAD"));
+    public static final ArrayList<String> METHOD_DESIGNATORS = new ArrayList<String>(
+            List.of("GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS"));
+
+    /* Annotations which make a resource method parameter a non entity parameter. */
+    public static final ArrayList<String> NON_ENTITY_PARAM_ANNOTATIONS = new ArrayList<String>(
+            List.of("FormParam", "MatrixParam", "QueryParam", "PathParam", "CookieParam", "HeaderParam", "Context"));
+
+    public static final String PATH_ANNOTATION = "Path";
 
     /* Source */
     public static final String DIAGNOSTIC_SOURCE = "jakarta-jax_rs";
 
     /* Severity */
     public static final DiagnosticSeverity SEVERITY = DiagnosticSeverity.Error;
```

Comment:
Let's remove this constant as it is redundant and call `DiagnosticSeverity.Error` directly in the `completeDiagnostic` method below